cli/tests/pcluster/config/test_section_cluster/test_cluster_from_file_to_cfn/pcluster.config.ini

```diff
@@ -148,8 +148,29 @@ ebs_settings = ebs1
 # Test: multiple ebs settings
 [cluster ebs-multiple]
 vpc_settings = default
+shared_dir = /work
 ebs_settings = ebs1,ebs2
```

Comment:
what happens when multiple EBS sections are used but one of them does not declare shared_dir? Can we add a test for this?

Reply:
That should be caught with `ebs_settings_validator`, I will look today to see how we can test this.